pkg/config/config.go

```diff
@@ -104,6 +104,9 @@ type AuthConfig struct {
 	// IdentityToken is used to authenticate the user and get
 	// an access token for the registry.
 	IdentityToken string `toml:"identitytoken" json:"identitytoken"`
+	CAFile        string `toml:"cafile" json:"caFile"`
```

Comment:
1) I prefer having the TLSConfig in a separate struct, probably a new field in `Registry` struct to keep backward compatibility.
2) Need to add comment for public fields
3) Follow the convention, `ca_file`, `cert_file` and `key_file` for toml.

Reply:
Moved and commented the fields.